qutebrowser/commands/command.py

```diff
@@ -173,6 +176,9 @@ def _check_prerequisites(self, win_id):
                 "{}: Only available with {} "
                 "backend.".format(self.name, self.backend.name))
 
+        if count == 0:
+            raise cmdexc.PrerequisitesError("Zero count not allowed")
```

Comment:
Please add the command name at the beginning of the error message here, like with the other errors.